### ISSUE TITLE
[Request] Option to disable logging when devMode is true

### ISSUE THREAD

**Issue by akteigland** [fullstorydev/fullstory-browser-sdk#188]
We set devMode on certain environments to ensure we do not send test data to FullStory. However when we run our automated test suites, the logs are full of "FullStory is in dev mode and is not recording" messages - easily over 50% of our test logs are this message, which makes it very annoying to sift through. It would be very helpful if we could enable devMode without logging.

**Comment by amyvofs** [comment]
Hi there @akteigland! Thanks so much for sharing your feedback on the current devMode limitations. I'll be happy to share this with our team to take into consideration as an improvement.

**Comment by morleytatro** [comment]
Yeah, hate to pile on here, but these warnings are super intrusive/annoying if you're ever trying to log something out and your app has a bunch of FS calls in it. Just an option to suppress would be nice.

**Comment by hussnaindev** [comment]
Hey guys, I have started working on this. I think I can add an option to disable logs.
